tests.mk

```diff
@@ -61,17 +61,17 @@ vagrant_test:
 ### go tests
 test:
 	@echo "--> Running go test"
-	@go test -p 1 $(PACKAGES) -tags deadlock
+	@go test -p 1 $(PACKAGES) -tags 'deadlock memdb goleveldb'
```

Comment:
Why should we use `memdb/goleveldb` as `default`?

Reply:
As `line/tm-db` was upgraded to the latest version, `memdb`, `goleveldb` were changed not to be included by default, but this was inconvenient and recently added `memdb` and `goleveldb` back to the default values, eliminating the need for this tag. I'll erase it.

Reply:
Thanks for your explaining